extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -94,6 +114,29 @@ export class AmpAutoAds extends AMP.BaseElement {
       });
   }
 
+  /**
+   * Selects into the use responsive ads for sizing in auto ads experiment branch.
+   * @param {boolean} isResponsiveEnabled
+   * @return {boolean}
+   */
+  isInUseResponsiveForResponsiveExperimentBranch(isResponsiveEnabled) {
+    const experimentInfoMap = /** @type {!Object<string,
+        !../../../src/experiments.ExperimentInfo>} */ ({
+      [[RESPONSIVE_SIZING_EXP.branch]]: {
+        isTrafficEligible: () => isResponsiveEnabled,
+        branches: [
+          [RESPONSIVE_SIZING_EXP.control],
+          [RESPONSIVE_SIZING_EXP.experiment],
+        ],
+      },
+    });
+    randomlySelectUnsetExperiments(this.win, experimentInfoMap);
```

Comment:
randomlySelectUnsetExperiments returns a mapping of experiment names to selected branches so you can minify this to:

return randomlySelectUnsetExperiments(this.win, experimentInfoMap)[RESPONSIVE_SIZING_EXP.branch] ==
      RESPONSIVE_SIZING_EXP.experiment;